### DISCUSSION TITLE
Doubts about the code file structure

### DISCUSSION THREAD

**Discussion by ForestsKing** [PaddlePaddle/PaddleSpatial#37] [BaiduKDDCup2022] [answered]
What is the difference between the files in 'PaddleSpatial/apps/wpf_baseline_gru/' and 'PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/', and why are there two?

**Comment by ramber1836** [comment, marked as answer]
Please refer to the second one.
